libandroid-navigation/build.gradle

```diff
@@ -45,13 +45,11 @@ dependencies {
   // Mapbox Android Services
   api (dependenciesList.mapboxServices) {
     transitive = true
-    exclude module: 'mapbox-java-geojson'
-    exclude module: 'mapbox-java-core'
-    exclude module: 'mapbox-java-services'
```

Comment:
I think it would make sense to re-exclude these modules to prevent duplicate imports.  `mapboxServices` is the `2.2.9` version of our services library, which we are using for just telemetry at the moment.  `mapboxSdkServices` is the `3.0.0` of the library and includes the modules above.  Sorry for the confusion on this as we transition these libraries!

Reply:
Whoops- okay, they are now re-excluded. Thanks for clarifying!